notebook/static/notebook/js/outputarea.js

```diff
@@ -783,6 +787,18 @@ define([
         element.append(toinsert);
         return toinsert;
     };
+    
+    var append_mathml = function (mathml, md, element) {
+        /**
+         * This method cannot do the typesetting because the mathml first has to
+         * be on the page.
+         */
+        var type = 'application/mathml+xml';
+        var toinsert = this.create_output_subarea(md, "output_mathml", type);
+        toinsert.text(mathml);
```

Comment:
wouldn't `.text()` escape the tags ?  should we use html, and make sure to sanitize ?